src/CalcManager/CEngine/scicomm.cpp

```diff
@@ -643,7 +650,11 @@ void CCalcEngine::ProcessCommandWorker(OpCode wParam)
             }
             break;
         }
-
+        else
```

Comment:
We talked about _this specific_ aspect of your change this morning, and would like to revert it.  I realize that at the time you were making the change, @grochocki wrote this [comment](https://github.com/microsoft/calculator/issues/264#issuecomment-511076350) in the Issue:

> Based on the conversation here and after looking at what other calculators do for unary operators, I am marking this as approved and unlocking #513 so that hitting "=" will always repeat the last operation.

However, after we played with this change, we realize it doesn't make sense for the <kbd>+/-</kbd> button to benefit from this change.  In fact, I'll be opening up a separate issue shortly to track stopping it from adding `negate()` to the history at all, and simply have it change the sign of the current input/result.  We don't want the <kbd>+/-</kbd> button to be treated like an operator at all, but rather just like an input key, the same way the <kbd>.</kbd> button is.

Reply:
I'm fine with this change! I will update the PR later tonight